src/components/Entry/EntryTabs.js

```diff
@@ -1,49 +1,105 @@
 import React, { useState } from 'react'
+import toastr from 'toastr'
+import 'toastr/build/toastr.min.css'
 import { Container, Form, Col } from 'react-bootstrap'
-import LogoGrey from '../../assets/logo-grey.png'
+import LogoGrey from '../../assets/images/logo-grey.png'
+import Loader from '../../assets/images/loading-gif.gif'
 
-let EntryTabs = () => {
+let EntryTabs = ({ apiClient, logInFunc }) => {
     let [tabShow, setTabShow] = useState("left")
+    let [formError, setFormError] = useState({
+        logIn: "",
+        signUp: ""
+    });
+    let [buttonDisabled, setButtonDisabled] = useState({ login: false, signup: false })
+
+    let handleSignUp = (e) => {
+        e.preventDefault();
+        // reference for username & both passwords
+        let username = e.target.username.value
+        let password = e.target.password.value
+        let passwordCheck = e.target.passwordCheck.value
+        // check if both passwords are correct
+        if (password !== passwordCheck) {
+            setFormError({ signUp: "Your passwords don't match."})
+        } else {
+            setButtonDisabled({ signup: true })
+            // if both passwords are correct, send data to API and process response status
+            apiClient.signUp(username, password)
+                .then(response => {
+                    toastr.success(response.data.message)
+                    setTabShow("left")
+                    e.target.username.value = ""
+                    e.target.password.value = ""
+                    e.target.passwordCheck.value = ""
+                })
+                .catch(err => {
+                    alert(err) // look at handling errors better - require a response to output message
+                    console.error(err)
+                })
+                .finally(() => setButtonDisabled({ signup: false }))
+        }
+    }
+
+    let handleLogIn = (e) => {
+        e.preventDefault();
+        let username = e.target.username.value;
+        let password = e.target.password.value;
+        setButtonDisabled({ login: true })
+        apiClient.logIn(username, password)
+            .then(response => {
+                logInFunc(response.data.token)
+            })
+            .catch(err => {
+                // setFormError({ logIn: err })
```

Comment:
Why is this commented out?

Reply:
again this is due to error handling which we said we'd look at - this is how i want to handle errors by setting them as a message to output using state

Reply:
again this is due to error handling which we we'll look at - this is how i want to handle errors by setting them as a message to output using state